packer/grapl-service/image.pkr.hcl

```diff
@@ -0,0 +1,163 @@
+packer {
+  required_version = ">= 0.12.0"
+}
+
+variable "build_ami" {
+  description = "Whether or not to actually build an AMI. Set to `false` for doing testing"
+  type        = bool
+  default     = true
+}
+
+variable "ami_name_prefix" {
```

Comment:
I'm not sure "ami_name_prefix" is the best name for this

Reply:
i don't like service_name (service is just super overloaded in grapl), went with image_name